plugins/leemons-plugin-assignables/frontend/src/components/Assignment/components/GroupPicker/GroupPicker.js

```diff
@@ -19,7 +19,10 @@ const useGroupPickerStyles = createStyles((theme) => ({
     gap: theme.other.global.spacing.gap.lg,
   },
   content: {
-    marginLeft: theme.other.global.spacing.padding.xlg,
+    // marginLeft: theme.other.global.spacing.padding.xlg,
+  },
```

Comment:
If unused, don't leave it commented, also the class

Reply:
done